_posts/2024-02-02-Demo_Build_10.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Visually Overhauled Demo ~ Demo Build 10
+author: SDS
+date: 2024-02-02 18:00:00 +0800
+categories: [Patch Notes, Demo]
+tags: [update, B10]
+image:
+    image:
+    path: /assets/images/posts/build10/cover-image.png
+    alt: the cover image for the post containing the text, "Visual Improvements Demo Build 10"
+---
+
+## Introduction
+
+Hey everyone! Thank you so much for staying interested in our game even with the slow pace of updates in the last few months! We can promise you we are still working hard on the game and we have hopes that the game shouldn't be in development for much longer!
+
+Majority of our team is busy with university classes as well as working on the game! Both being huge undertakings so from the bottom of our hearts, we appreciate you sticking with us!
+
+Anyways, we know you guys are sick of hearing about updates for the demo! We are too! But we hope it won't be long before we can bring you the *real* thing!
+
+A quick not that we wanted to remind everyone of is that our goal with the demo is for it to reflect the game in a condensed way. With this information, know that changes made to the demo, often reflect changes that we may have made to the full game, often in a condensed way. We want to mention this specifically because of the fact that if you like something in the demo, we can assure you we will expand on it and maximize it to the fullest potential in the full game.
+
+Anyways, enough of that! Let's dive into the patch notes!
+
+## New Trailer
+
+{% include embed/youtube.html id='QR7BFpFaarg' %}
+
+## Changelog
+
+We have pushed out an update to our demo branch which include the following *specified* changes:
+
+**Added Post Processing Effects**
+: When working on improvements to visuals, post processing effects often are included. We hope to bring an optimized and realistic feeling post processing set to the game's visuals.
+
+**Improved Lighting Effects**
+: Good lighting changes help make the player feel like they are truly in the environment that we hope to immerse them in. We hope that these changes to the way that the lighting system works will improve the look and feel of the game!
+
+**Improved Environmental Details**
+: We improved some of the environmental details that were previously added to our game along with introducing new ones such as dust particles to once again improve the feel of the game.
+
+**Improved Flashlight Visuals**
+: We made various changes to the flashlight and the way it interacts with the world around it. We thought this was an important change as the flashlight is one of the key features of our game.
```

Comment:
```suggestion
: We made various changes to the flashlight and the way it interacts with the world around it. We thought this was an important change as the flashlight is one of the key features of our game.

"We made various changes to the flashlight and the way it interacts with the world around it. We thought this was an important change as the flashlight is one of the key features of our game."

I'd say we improved the look of the flashlight by adding some additional details instead because it functions the same in the demo. Nothing functionally changes in the demo for the flashlight
```